app/src/main/java/com/cmput301f20t21/bookfriends/ui/add/AddEditActivity.java

```diff
@@ -287,14 +277,29 @@ private void onFailure(BOOK_ERROR error) {
      * @param uri the local image uri
      */
     private void paintImage(Uri uri) {
-        GlideApp.with(this)
-                .load(uri)
-                .diskCacheStrategy(DiskCacheStrategy.NONE)
-                .skipMemoryCache(true)
-                .placeholder(R.drawable.no_image)
-                .into(bookImage);
+        if (uri == null) {
+            GlideApp.with(this)
+                    .load(R.drawable.no_image)
+                    .into(bookImage);
+        } else {
+            GlideApp.with(this)
+                    .load(uri)
+                    .placeholder(R.drawable.no_image)
+                    .into(bookImage);
+        }
+    }
+    private void paintImage(String uri) {
+        if (uri == null) {
+            GlideApp.with(this)
+                    .load(R.drawable.no_image)
+                    .into(bookImage);
+        } else {
+            GlideApp.with(this)
+                    .load(uri)
+                    .placeholder(R.drawable.no_image)
+                    .into(bookImage);
+        }
```

Comment:
found something interesting as I was thinking how to not have Glide/GlideApp in so many fragments/activities, the Glide.with() part actually accepts the view as argument(even imageView!), so I tried the below code and it actually works:

![image](https://user-images.githubusercontent.com/43459469/99314468-66936e00-281e-11eb-8f9f-04f01a9dac30.png)

If this actually work everywhere then I'm thinking we can have a util/helper class like this and everytime we need to load an image we just call the helper class instead of having Glide/GlideApp everywhere in our app

But this will be another PR, I will stash my changes and maybe do a small demo tonight to discuss